.tekton/scanner-v4-pipeline.yaml

```
@@ -0,0 +1,372 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: scanner-v4
+spec:
+
+  finally:
+  - name: show-sbom
+    params:
+    - name: IMAGE_URL
+      value: $(tasks.build-container.results.IMAGE_URL)
+    taskRef:
+      params:
+      - name: name
+        value: show-sbom
+      - name: bundle
+        value: quay.io/redhat-appstudio-tekton-catalog/task-show-sbom:0.1@sha256:1f90faefa39c2e4965793c1d8321e7d5d99a6c941276a9094a4e0d483a598fca
+      - name: kind
+        value: task
+      resolver: bundles
+  - name: show-summary
+    params:
+    - name: pipelinerun-name
+      value: $(context.pipelineRun.name)
+    - name: git-url
+      value: $(tasks.clone-repository.results.url)?rev=$(tasks.clone-repository.results.commit)
+    - name: image-url
+      value: $(params.output-image)
+    - name: build-task-status
+      value: $(tasks.build-container.status)
+    taskRef:
```

Somehow `show-summary`'s workspace got lost. The task did not fail because it's marked as optional there.
```suggestion
    workspaces:
      - name: workspace
        workspace: workspace
    taskRef:
```